packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -120,6 +119,13 @@ export class MLSService extends TypedEventEmitter<Events> {
       this.config.nbKeyPackages,
     );
 
+    await this.coreCryptoClient.registerCallbacks({
+      // All authorization/membership rules are enforced on backend
+      clientIsExistingGroupUser: async () => true,
+      authorize: async () => true,
+      userAuthorize: async () => true,
```

Comment:
These were previously provided by webapp, but since the rules are enforced by backend anyway, we can move them to the core.